src/Uno.UI/Extensions/ViewHelper.Android.cs

```diff
@@ -107,6 +107,12 @@ public static void RefreshFontScale()
 		{
 			using (Android.Util.DisplayMetrics displayMetrics = Android.App.Application.Context.Resources.DisplayMetrics)
 			{
+				if (FeatureConfiguration.Font.IgnoreTextScaleFactor)
+				{
+					// Keep disabling text scaling
+					displayMetrics.ScaledDensity = displayMetrics.Density;
+				}
```

Comment:
This is only doing parts of the work that is done here: 

https://github.com/unoplatform/uno/blob/7e80735642b9f0ed76f54452964afcce7ef1a0d1/src/Uno.UI/Extensions/ViewHelper.Android.cs#L79-L96

We should probably refresh the whole set instead of just those two (and keep the comment that is very insightful).

Reply:
What specifically do you think I should add? 
because `_cachedScaledDensity` will be set right after my changes and I don't see why we should refresh the whole set

Reply:
I mean the comment is insightful, it's worth adding. 
But in my opinion `displayMetrics.Density` and `displayMetrics.Xdpi` are not related to the `RefreshFontScale` method, are they?

Reply:
Agreed, the rest of the method does not seem impacted. In that case, let's extract the portion that is the same in both methods to avoid duplication.